cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -119,13 +128,53 @@ module postgreServer 'br/public:avm/res/db-for-postgre-sql/flexible-server:0.1.4
     ]
     databases: [
       {
-        name: databaseName
+        name: postgreSqlDatabaseName
+      }
+    ]
+    location: location
+    {{- if (and .DbPostgres .DbPostgres.AuthUsingManagedIdentity) }}
+    roleAssignments: [
+      {
+        principalId: connectionCreatorIdentity.outputs.principalId
+        principalType: 'ServicePrincipal'
+        roleDefinitionIdOrName: 'b24988ac-6180-42a0-ab88-20f7382dd24c'
```

Comment:
This role is added in 4 scopes: 
1. Azure Container Apps Environment.
2. Azure Container Apps
3. Azure Container Apps managed identity
4. Azure Database for PostgreSQL.

Delete any one of above 4 scopes will cause `az containerapp connection` fail.

Reply:
why do we need ACA Environment here?

Reply:
Without ACA environment, it will return error like `Can not read xxx in environment xxx`.